### ISSUE TITLE
[Select] Placeholder not visible when dropdown is open

### ISSUE THREAD

**Issue by ah-arch** [telus/tds-core#629]
## Description

* The **Chrome** behavior for the TDS Select component using the default HTML `<select />` with a `<option hidden disabled />` as placeholder hides the placeholder entirely when the dropdown is open. This also affects **Edge** and **Firefox**, but not **IE** or **Safari**.
* Additionally on **Chrome**, the default checkmark appears on the first available option as a result
* Removing the `hidden` prop would show the placeholder which
    * Fixes the misleading checkmark
    * Keeps it more consistent across browsers
* Perhaps it was put in to hide the placeholder on mobile when dropdown is open? Not sure if that is intended accessibility or UX.

## Reproduction Steps

* Open [TDS Select component](https://tds.telus.digital/components/index.html#select) example on Chrome
* Click on the `<select/>` to trigger the dropdown to appear
* Observe that:
    * there is no disabled placeholder, and
    * the default checkmark appears on the first available option

## Reproduction Percentage (if applicable)

* 10/10 times

## Meta

* TDS version: Split component - [Select v1.0.6](https://github.com/telusdigital/tds-core/blob/b937a2e88e0ba52892bc808a62370088038d0363/packages/Select/Select.jsx#L30)
* Willing to develop solution: Yes
* Has workaround: No
* High impact: No

### Safari
<img width="538" alt="safari" src="https://user-images.githubusercontent.com/4450690/40011888-23eac514-5777-11e8-9835-be39b115b8c0.png">

### IE
![ie](https://user-images.githubusercontent.com/4450690/40011897-284053d6-5777-11e8-99b1-4c2a453829df.png)

### Firefox
<img width="789" alt="firefox" src="https://user-images.githubusercontent.com/4450690/40011925-36ba862a-5777-11e8-9709-bf7201661ff7.png">

### Edge (Placeholder is not visible)
![edge](https://user-images.githubusercontent.com/4450690/40011962-4d6fab84-5777-11e8-9bd9-d3e1d3fc140d.png)

### Chrome (Placeholder is not visible, and first option has checkmark)
![chrome - select with hidden on placeholder](https://user-images.githubusercontent.com/4450690/40011991-5f09395a-5777-11e8-9ad9-4bfadfae81f2.png)
* Mobile (Placeholder is not visible)
    ![chrome - mobile](https://user-images.githubusercontent.com/4450690/40012429-a66f1b56-5778-11e8-8b9f-42d39c4dce56.png)

## Screenshots of Suggested Solution
Using `<option disabled />`
### Chrome
![chrome - select without hidden on placeholder](https://user-images.githubusercontent.com/4450690/40012611-32e88400-5779-11e8-8139-f0c4385e8de1.png)
* Mobile
    ![chrome - mobile - without hidden](https://user-images.githubusercontent.com/4450690/40012673-5ba7b690-5779-11e8-8c2f-b2bdcb4e0123.png)

**Comment by theetrain** [comment]
Hi @ah-arch,

Thanks for making this suggestion. If you have cycles to submit a pull request to amend this issue, we would greatly appreciate it.

**Comment by ah-arch** [comment]
It is not high priority, and it is a one-word change if the suggested solution is warranted.

Attempted to [run](https://tds.telus.digital/contributing/developer-guide.html) `yarn bootstrap`, `yarn test` and `yarn dev` after installing but they all failed with _modules_ / _import_ errors. So probably not _cycling_ anytime soon.

**Comment by jraff** [comment]
Available in [@tds/core-select@1.0.11](https://github.com/telus/tds-core/compare/@tds/core-select@1.0.10...@tds/core-select@1.0.11)
